Configurations/windows-makefile.tmpl

```diff
@@ -588,8 +588,8 @@ reconfigure reconf:
 	  return <<"EOF";
 $args{src}: $args{generator}->[0]
 	pod2html "--podroot=\$(SRCDIR)/doc" --htmldir=.. \\
-		 --podpath=man1:man3:man5:man7 "--infile=\$<" "--title=$title" \\
-	| \$(PERL) -pe "s|href=\\"http://man\\.he\\.net/(man\d/[^\\"]+)(?:\\.html)?\\"|href=\\"../\$1.html|g;" \\
+		 --podpath=man1:man3:man5:man7 "--infile=\$?" "--title=$title" \\
+	| \$(PERL) -pe ^"s^|href=\\^"http://man\\.he\\.net/^(man\\d/[^^\\^"]+^)^(?:\.html^)?^"^|href=\\^"../\$\$1.html^|g;^" \\
```

Comment:
Here an explanation of the tricky parts
* The `^` characters deactivate all special cmd.exe characters, in particular `^"` deactives the inside-string-mode of cmd.exe entirely.
* The `\\^"` is passed as `\"` to perl.exe, where it gets converted to `"` by CommandLineToArgvW().
* `../\$\$1.html`  is seen by make as `../$$1.html` and the double $$ prevents it from expanding `../$1.html` to `../.html`.
* `[^^\\^"]` is passed as  `[^\"]`  from cmd.exe to perl and ends up as `[^"]` when parsed by CommandLineToArgvW().

Reply:
It [feels like a victory](https://en.wikipedia.org/wiki/Pyrrhic_victory) over cmd.exe!

Reply:
I have reviewed the changes and they seem OK to me, but I have not formally approved because I have close to 0 experience on Windows-specific development and I wonder if the special `^` escape of quotes and `^` itself is compatible across all the supported windows platforms.

Reply:
`^` has been part of the cmd.exe syntax for a while, so I consider it fairly safe.